build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -2000,6 +2188,42 @@ message Compressor {
   }
 }
 
+// Content-defined chunking algorithms used for splitting blobs into chunks.
+message ChunkingAlgorithm {
+  enum Value {
+    // No special constraints about the used chunking algorithm. If a client
+    // selects this value this means he does not care about which exact chunking
+    // algorithm is used. Thus, the server is free to use any chunking algorithm
+    // at its disposal. A server does not need to advertise this value.
+    DEFAULT = 0;
+
+    // Content-defined chunking using Rabin fingerprints. An implementation of
+    // this scheme in presented in this paper
+    // https://link.springer.com/chapter/10.1007/978-1-4613-9323-8_11. The final
```

Comment:
This document is behind a paywall. Any chance we can link to a spec that is freely accessible?

Reply:
Actually, the paper about FastCDC https://ieeexplore.ieee.org/document/9055082 also contains an algorithmic description of the Rabin fingerprinting technique (Algorithm 3. RabinCDC8KB). The only thing that is missing here is a precise description of the precomputed U and T arrays. Even though, they provide links to other papers, where these arrays are supposedly defined, I could not find any definition in these papers.